core/rfb.js

```diff
@@ -46,7 +46,6 @@ const DEFAULT_BACKGROUND = 'rgb(40, 40, 40)';
 var _videoQuality =  2;
 var _enableWebP = false;
 var _enableQOI = false;
-var _clickEligible = false;
```

Comment:
@thelamer This was never used as it's outside the class, (the 3 above it are never used either). So what was happening in you code was the first time `this. _clickEligible` was called it would return undefined, then you would later set a local class variable that didn't exist before. I'm not sure if this contributed to it not working in MacOS, but interesting nonetheless for future reference. The class variables are set around line 100 onwards.